### PR TITLE
deposit: small fronted fixes

### PR DESCRIPTION
**Pull request by crepererum**
Here are multiple small fronted fixes for the deposit module which I encountered while working on the analysis preservation code.

### CODE REVIEW

File: invenio/modules/deposit/static/css/deposit/form.css [hunk 1]
```diff
@@ -64,6 +64,32 @@ span.ui-icon.ui-icon-circle-triangle-e {
   display:none;
 }
 
+.hide-on-collapsed{
+	display:block
```
Comment (jirikuncar) [inline invenio/modules/deposit/static/css/deposit/form.css]: indentation ...

File: invenio/modules/deposit/static/css/deposit/form.css [hunk 2]
```diff
@@ -64,6 +64,32 @@ span.ui-icon.ui-icon-circle-triangle-e {
...
+}
+
+.panel-toggle.collapsed .hide-on-collapsed,.show-on-collapsed{
```
Comment (jirikuncar) [inline invenio/modules/deposit/static/css/deposit/form.css]: space after `,`

File: invenio/modules/deposit/templates/deposit/widget_plupload_base.html [hunk 3]
```diff
@@ -50,7 +50,7 @@
     <tbody id="sortable">
     </tbody>
   </table>
-  <p class="text-muted" align="center"><small>Drag and drop files to upload</small></p>
+  <p id="field-plupload_file" class="text-muted" align="center"><small>Drag and drop files to upload</small></p>
```
Comment (jirikuncar) [inline invenio/modules/deposit/templates/deposit/widget_plupload_base.html]: Is it possible to avoid using `id`?

Comment (crepererum) [inline invenio/modules/deposit/templates/deposit/widget_plupload_base.html]: I've only added the id that is requested by some JavaScript code which is not mine. If the code isn't there, it seems to fail (JS error).

Comment (jirikuncar) [inline invenio/modules/deposit/templates/deposit/widget_plupload_base.html]: ok

**Comment by crepererum** [comment]
Ready.

File: invenio/modules/deposit/templates/deposit/run_base.html [hunk 4]
```diff
@@ -47,18 +47,17 @@
 {% endblock form_submit_dialog %}
 {% block deposit_page %}
 <div class="row">
+        <div class="col-md-3 col-md-push-9">
```
Comment (jirikuncar) [inline invenio/modules/deposit/templates/deposit/run_base.html]: indentattion

**Comment by crepererum** [comment]
Ready v2.

**Pull request merged**